### PR TITLE
Christmas break 2024

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: content/issues/2024-12-23-christmas-2024.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+date: "2024-12-24T19:00:00+01:00"
+resolved: false
+#resolvedWhen: "2025-01-02T09:00:00+01:00"
+section: issue
+severity: notice
+---
+
+Starting from 24th of December all the team will be on vacation.
+
+Give us some time to get refreshed and be better prepared to work hard on Packit's new features and bugs.
```
Comment (lbarcziova) [inline content/issues/2024-12-23-christmas-2024.md:12]:
```suggestion
Please, give us some time to get refreshed and be better prepared to work hard on Packit's new features and bugs! We should be back on January 2nd.
```

**Review by lbarcziova — APPROVED** [review]
thanks!

File: content/issues/2024-12-23-christmas-2024.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Christmas is here 🎄"
+date: "2024-12-24T19:00:00+01:00"
```
Comment (nforro) [inline content/issues/2024-12-23-christmas-2024.md:3]: Will somebody really be working until 7pm on 24th? :sweat_smile:

Comment (mfocko) [reply content/issues/2024-12-23-christmas-2024.md:3]: there's a different date in the filename though :eyes:

Comment (majamassarini) [reply content/issues/2024-12-23-christmas-2024.md:3]: initially it was 23T19 but then I thought 24 was better and I forgot to change the hour... thanks for spotting it!

**Pull request merged**